Resources/doc/reference/getting_started.rst

```diff
@@ -0,0 +1,105 @@
+Getting Started
+===============
+
+The bundle works on top of 2 simple models :
+ * an ``Article``\ : An article is composed of Fragments and contains information about a content
+   (status, type, publication date, categories, etc...)
+ * a ``Fragment``\ : A fragment contains information about a part of the full content. Each fragment got his own template.
+
+
+Render an article
+-----------------
+
+**This part is currently in development.**
+The goal is provide a simple Controller and Dynamic route who will fetch an article, then if all checks succeed, display it for the user.
+
+.. code-block:: php
+
+    /**
+     * Article index action
+     *
+     * @param Request $request
+     * @param string  $id
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     *
+     * @throws \Symfony\Component\HttpKernel\Exception\NotFoundHttpException
+     */
+    public function indexAction(Request $request, $id)
+    {
+        /** @var Page $page */
+        $page    = $this->getPage($request);
+        $article = $this->entityManager->find(Article::class, $id);
+
+        if (!$article) {
+            throw new NotFoundHttpException(sprintf('No article found for ID "%s"', $id));
```

Comment:
I'm still working on it, I've got a cooworker who told me another way to do this whole action, will look into it. For the moment I left this part of code just as information, but it will be remove from the doc (or move somewhere else, it just not belong here i think)